app/background.cc

```diff
@@ -191,7 +191,10 @@ int grab_background(std::shared_ptr<background_t> pbkd, int width, int height, c
     } else {
         // resize still image as requested into out
         cv::Rect crop = calcCropping(pbkd->raw.cols, pbkd->raw.rows, width, height);
-        cv::resize(pbkd->raw(crop), pbkd->raw, cv::Size(width, height));
+		// under some circumstances we must do the job with 2 step!
+		// without this resize(pbkd->raw(crop), pbkd->raw, ...) may fail.
```

Comment:
```suggestion
	// Under some circumstances we must do the job in two steps!
	// Otherwise this resize(pbkd->raw(crop), pbkd->raw, ...) may fail.
```

Reply:
I have pushed the suggestion and also try to reproduce the same problem for the video as with the still image for the video part. I can produce a crash only on very strange resolutions for the video and the output size, and the code within background what never involved. So I expect that a normal user will not have issue with the actual code.